labels.md

```diff
@@ -0,0 +1,220 @@
+# Goals
+
+* synchronise labels between repos
+* make labels easier to set automatically
+* make labels clearer
+* make it easier to tell at a glance what is blocking PRs from landing
+* make colours more useful by making them more uniform
+
+# Naming scheme
+
+## Current issues
+
+* TiDB uses a `category/label` scheme, but with some exceptions
+  - closed, component, priority, status, type
+* TiKV uses a `C: label` scheme
+  - C (component), D (difficulty), P (priority), S (status), T (type)
+* PD uses a `category/label` scheme, but with some exceptions
+  - area, backport, kind, priority, 
+
+Bots work on all repos, but bot-readable labels are not distinguished.
+
+There is some duplication within a repo and some inconsistency about the category a label belongs to.
+
+## Proposal
+
+We use a `C: label` scheme for all repos (I don't feel strongly about the scheme, as long as there is just one. `C: label` is shorter, `category/label` is easier to understand). The categories are: component, priority, status, type, and difficulty.
+
+Where labels are similar across repos we pick one name and use that. For the status labels we use `waiting on CI`, `waiting on review`, `waiting on discussion`, and `waiting on author`, these are taken from the Rust repo where they have been found to be useful. We'll also keep some of the existing statuses where they are useful. The component labels are unique to each repo.
+
+# Colours
+
+Are generally inconsistent.
+
+## Proposal
+
+All labels in the same category have the same colour, taking the colour which is currently most popular.
+
+
+# Aternatives
+
+* Use `category/label`, `cat/label`, or `C-label` naming schemes
+* More categories (e.g., a bot category)
+* Allow labels without categories
+* Add emoji to highlight some labels or sub-categories
+* Separate labels for issues vs PRs
+
+
+# The labels
+
+`*` indicates I think we should remove an issue, there might be a suggested replacement in `<...>`. `?` means I don't know what the label means.
+
+Current labels and colours are on the left, proposed are on the right. Any non-C labels which are present in any repo will be added to all repos.
+
+## TiDB
+
+```
+CHANGELOG                       #a206c9     type/CHANGELOG                      #1d76db
+closed/no-pr-quota              #b60205     status/Closed pr-quote              #e6e6e6
+closed/outdated                 #b60205     status/Closed outdated              #e6e6e6
+component/DDL-need-LGT3         #00bff4     component/DDL (3lgtm)               #d1fad7
+component/GC                    #cce2ff     component/GC                        #d1fad7
+component/binlog                #8befc9     component/Binlog                    #d1fad7
+component/build                 #f2b87b     component/Build                     #d1fad7
```

Comment:
Can you provide some logic or reasoning why the `lowercase/Camelcase lowercase lowercase` is an improvement over `lowercase/lowercase-lowercase` that was previously used here?

Reply:
No :-s, the casing/spacing was inconsistent between labels so I just picked one and used it everywhere. Originally this was, e.g., `C: Binlog`, I think that after changing the naming scheme it might be better to use `component/binlog` (but `component/foo bar` rather than `component/foo-bar`)